docs/user/requirements.rst

```diff
@@ -20,16 +20,15 @@ The following packages are needed in order to run RocketPy:
 - Numpy >= 1.0
 - Scipy >= 1.0
 - Matplotlib >= 3.0
-- netCDF4 >= 1.4, < 1.6 for Python 3.7+, netCDF4 >= 1.6.2 for Python 3.11
+- netCDF4 >= 1.6.4
```

Comment:
I recall hearing that this version of `netCDF4` solves the issues that we were having, however I did not verify this. Hence, may I ask if you were able to check whether the `collab` and `3.11` install work on `1.6.4`?

Reply:
Yes, ot is running perfectly.